src/mame/drivers/cps1.cpp

```diff
@@ -1962,16 +1962,25 @@ INPUT_PORTS_START( sf2 )
 	PORT_DIPSETTING(    0x00, DEF_STR( Test ) )
 INPUT_PORTS_END
 
-/* Needs further checking */
 static INPUT_PORTS_START( sf2j )
 	PORT_INCLUDE( sf2 )
 
 	PORT_MODIFY("DSWB")
 	PORT_DIPNAME( 0x08, 0x00, "2 Players Game" )                    PORT_DIPLOCATION("SW(B):4")
-	PORT_DIPSETTING(    0x08, "1 Credit/No Continue" )
+	PORT_DIPSETTING(    0x08, "1 Credit/No Continue" ) // 1 Credit for versus play, and "Here comes a new challenger" option is disabled
 	PORT_DIPSETTING(    0x00, "2 Credits/Winner Continue" ) //Winner stays, loser pays, in other words.
 INPUT_PORTS_END
 
+/* Same as sf2j but option are inverted. Default position here is 2 Credits for versus play */
+static INPUT_PORTS_START( sf2cej )
+	PORT_INCLUDE( sf2 )
+
+	PORT_MODIFY("DSWB")
+	PORT_DIPNAME( 0x08, 0x00, "2 Players Game" )                    PORT_DIPLOCATION("SW(B):4")
```

Comment:
It should probably still default to having this switch off, even if it has a different effect.  It seems Capcom wanted to change the default behaviour.

Reply:
Yes, you're right. I forgot to set the default position in PORT_DIPNAME macro. Now fixed.